### ISSUE TITLE
Guake snap package

### ISSUE THREAD

**Issue by robert-matusewicz** [Guake/guake#1677]
Hi,

Last few days I was working on creating a snap package for Guake and today I finally was able to create a first version that works. In few days, I plan to push that "alpha" version of package to the SnapCraft - the name that I want to use is "guake". Because I'm not a developer of the project, I want to use a process that is described here: https://forum.snapcraft.io/t/join-snapcrafters/1325 and here: https://github.com/snapcrafters/fork-and-rename-me.

One of the last step is snap adoption by the project maintainers/developers - so, my question is: are you interested in having a snap package for guake and are you willing to create an official snapcraft account where snap will be stored? I'm more then willing to tackle all the work that is required to create the package and maintain it in the future, but I alone cannot make that package "official".

Btw, it you want to test the guake snap, the current version is here: https://github.com/robert-matusewicz/guake/tree/snap

In order to build it just clone the branch, cd into it and run

snapcraft

and that will create a package

**Comment by gsemet** [comment]
Hi. Looks like a great idea, so I do not see any problem into merging your fork, and ideally have the travis pipeline publish the official snap at the end of its build (like it does for the pip package). I don't know how publication works, but if you have done pretty much all the work, I do not see why the Guake community would not benefit from it.

Maybe we can move the snap think to a dedicated project you would be the main maintainer, something like https://github.com/Guake/guake-snap/ with only the code for the snap packaging.

Basically, tell me what should I do to integrate your packaging and have it automatically published and I can do it. Where should I create the snap account and do you know if it is possible to delegate the publication permission to you easily?

**Comment by robert-matusewicz** [comment]
I think creating a guake-snap project and making me maintainer is a good idea. I already registered guake name in snapcraft, so I will not need to go through the process of passing ownership o package to snapcrafters and then back to Guake!

**Comment by gsemet** [comment]
Here we go: https://github.com/Guake/guake-snap

You are maintainer on this project. Just tell me if you need some kind of help, settings deploy keys, setting up travis somehow,...

**Comment by robert-matusewicz** [comment]
Thanks - as soon as I will have a first version published to the store, I will ping you for some help.

**Comment by robert-matusewicz** [comment]
Oh my - I encountered a complication - the gnome extension that is the preffered way to created snaps for Gnome applications works only with strictly confined snaps. That will not work for Gnome. I need to figure out what to do with this (obviously, I can build mu own gnome extension that works in classic, but I'm not sure if that will pass store review)

**Comment by Rakly3** [comment]
`sudo snap install guake
error: snap "guake" is not available on stable but is available to install on the following
       channels:

       edge       snap install --edge guake

       Please be mindful pre-release channels may include features not completely tested or
       implemented. Get more information with 'snap info guake'.
`
